packages/charts/src/charts/donut/donut-total.component.tsx

```diff
@@ -3,7 +3,8 @@ import React, { FC, memo } from 'react';
 import { Typography } from '@keen.io/ui-core';
 
 type Props = {
-  children: string | number;
+  /* Reach children nodes */
```

Comment:
typo `React children nodes` or remove comment as other props are not described

Reply:
Fixed :) Thanks !